cfdilib/templates/cfdv33.xml

```diff
@@ -78,6 +78,12 @@
                     {% endif %}
                 </cfdi:Impuestos>
             {% endif %}
+            {% if line.customs_number %}
```

Comment:
I think that this if could be removed,

Maybe with `{% for value in line.customs_number or []%}`